rust/aes-gcm-siv/src/error.rs

```diff
@@ -3,27 +3,16 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 //
 
-use std::fmt;
-
-#[derive(Clone, Debug, Eq, PartialEq)]
+#[derive(thiserror::Error, Clone, Debug, Eq, PartialEq)]
```

Comment:
This causes `Error` to implement `std::error::Error`. This _is_ a behavioral change, as this error type did not implement the std error trait before this commit.